docker/backend.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:10-alpine AS build
+RUN apk add git automake autoconf build-base zlib-dev
+
+USER node
+WORKDIR /home/node
+RUN git clone --depth=1 https://github.com/kuzn-ilya/ryazan.git
+
+WORKDIR /home/node/ryazan/server
+RUN sed -i 's/127.0.0.1/database/g' "./config/environments/development/database.json"
```

Comment:
can't see this config anywhere 😅

Reply:
It's a strapi config. and this is a workaround. strapi development config can only connect to `localhost` but `database` host is needed. I didn't want to change anything.